src/lib/components/progress/QCircularProgress.svelte

```diff
@@ -33,7 +29,16 @@
     children = fallback,
     ...props
   }: QCircularProgressProps = $props();
+  // #endregion: --- Props
+
+  // #region:    --- Non-reactive variables
+  const radius = 50,
+    diameter = 2 * radius,
+    circumference = diameter * Math.PI,
+    strokeDashArray = Math.round(circumference * 1000) / 1000;
```

Comment:
```suggestion
  const radius = 50;
  const diameter = 2 * radius;
  const circumference = diameter * Math.PI;
  const strokeDashArray = Math.round(circumference * 1000) / 1000;
```
Both styles are valid, this is just a suggestion to stay consistent. Or do we use comma initialization often?

Reply:
I don't have a strong opinion on this, I can go with your code